fontir/src/variations.rs

```diff
@@ -29,6 +30,10 @@ const ONE: OrderedFloat<f32> = OrderedFloat(1.0);
 pub struct VariationModel {
     default: NormalizedLocation,
 
+    /// Non-point axes
+    axes: Vec<Axis>,
+    axis_names: HashSet<String>,
```

Comment:
I will continue to register my dislike for storing axis names as `String` 🤷

Reply:
Resolving as I would like to tackle that in it's own PR